src/parse/asp/config.go

```diff
@@ -108,10 +108,19 @@ func resolvePluginValue(values []string, subrepo string) []string {
 				continue // I guess it wasn't a build label. Leave it alone.
 			}
 			// Force the full build label including empty subrepo so this is portable
-			v = fmt.Sprintf("///%v//%v:%v", l.Subrepo, l.PackageName, l.Name)
+			var buf strings.Builder
```

Comment:
Add a comment here (or at the top of the file) to explain why you're not using fmt.Sprintf